### PR TITLE
ci: prepare actions for dev branch

### PR DESCRIPTION
**Pull request by Yeti-or**
Настроил github actions на ветку `dev`
perfomance тесты пришлось дублировать

:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.521.5056692110.swift)  
[color_sberHealth_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.521.5056692110.kt)  
[color_sberHealth_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.521.5056692110.ts)  
[color_sbermarket_business_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.521.5056692110.swift)  
[color_sbermarket_business_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.521.5056692110.kt)  
[color_sbermarket_business_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.521.5056692110.ts)  
[color_sbermarket_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.521.5056692110.swift)  
[color_sbermarket_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.521.5056692110.kt)  
[color_sbermarket_metro_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.521.5056692110.swift)  
[color_sbermarket_metro_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.521.5056692110.kt)  
[color_sbermarket_metro_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.521.5056692110.ts)  
[color_sbermarket_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.521.5056692110.ts)  
[color_sbermarket_selgros_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.521.5056692110.swift)  
[color_sbermarket_selgros_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.521.5056692110.kt)  
[color_sbermarket_selgros_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.521.5056692110.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.521.5056692110.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.521.5056692110.kt)  
[color_sbermarket_wlbusiness_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.521.5056692110.ts)  
[color_sberonline_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.521.5056692110.swift)  
[color_sberonline_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.521.5056692110.kt)  
[color_sberonline_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.521.5056692110.ts)  
[color_sberprime_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.521.5056692110.swift)  
[color_sberprime_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.521.5056692110.kt)  
[color_sberprime_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.521.5056692110.ts)  
[shadow_sbermarket_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.521.5056692110.ts)  
[typo_mage_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.521.5056692110.swift)  
[typo_mage_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.521.5056692110.kt)  
[typo_mage_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.521.5056692110.ts)  
[typo_plasma_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.521.5056692110.swift)  
[typo_plasma_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.521.5056692110.kt)  
[typo_plasma_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.521.5056692110.ts)  
[typo_ruler_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.521.5056692110.swift)  
[typo_ruler_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.521.5056692110.kt)  
[typo_ruler_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.521.5056692110.ts)  
[typo_sage_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.521.5056692110.swift)  
[typo_sage_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.521.5056692110.kt)  
[typo_sage_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.521.5056692110.ts)  
[typo_sbermarket_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.521.5056692110.swift)  
[typo_sbermarket_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.521.5056692110.kt)  
[typo_sbermarket_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.521.5056692110.ts)  
[typo_soulmate_ios-swift--canary.521.5056692110.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.521.5056692110.swift)  
[typo_soulmate_kotlin--canary.521.5056692110.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.521.5056692110.kt)  
[typo_soulmate_react-native--canary.521.5056692110.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.521.5056692110.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.215.1-canary.521.5056692110.0
  npm install @salutejs/plasma-temple@1.164.1-canary.521.5056692110.0
  npm install @salutejs/plasma-ui@1.196.1-canary.521.5056692110.0
  npm install @salutejs/plasma-web@1.215.1-canary.521.5056692110.0
  # or 
  yarn add @salutejs/plasma-b2c@1.215.1-canary.521.5056692110.0
  yarn add @salutejs/plasma-temple@1.164.1-canary.521.5056692110.0
  yarn add @salutejs/plasma-ui@1.196.1-canary.521.5056692110.0
  yarn add @salutejs/plasma-web@1.215.1-canary.521.5056692110.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-521/

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-521/

**Review by neretin-trike — APPROVED** [review]

File: .github/workflows/performance-test-dev.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
+name: Component Performance Testing Dev
```
Comment (akhdrv) [inline .github/workflows/performance-test-dev.yml:1]: а зачем копипастить?

Comment (akhdrv) [reply .github/workflows/performance-test-dev.yml:1]: аа, типа с девом сравнивать

Comment (akhdrv) [reply .github/workflows/performance-test-dev.yml:1]: давай может тогда не копипастить, а интерполировать везде ```${{github.event.pull_request.base.ref}}```

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-521/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-521/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-521/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-521/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-521/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-521/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-521/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: .github/workflows/performance-test.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
 
       - uses: actions/checkout@v2
         with:
-          ref: master
+          ref: ${{github.event.pull_request.base.ref}}
           clean: false
 
       - name: Prepare repository for master
```
[the remark this answers is not in the document]
Comment (Yakutoc) [reply .github/workflows/performance-test.yml:52]: > вот этот вот шаг вроде нигде кроме релиза не нужен?

@akhdrv привет, пожалуйста объясни почему так считаешь?  Я как раз изучаю/ищу возможные пути ускорения нашего CI

[the remark this answers is not in the document]
Comment (akhdrv) [reply .github/workflows/performance-test.yml:52]: конкретно это место проверю тут https://github.com/salute-developers/plasma/pull/519

[the remark this answers is not in the document]
Comment (akhdrv) [reply .github/workflows/performance-test.yml:52]: да, все норм без него https://github.com/salute-developers/plasma/actions/runs/5067885967/jobs/9099489221?pr=519

**Pull request merged**